app/views/my_facilities/drug_stocks/_drug_stocks_table.html.erb

```diff
@@ -51,17 +60,19 @@
           <% end %>
         <% end %>
 
-        <% if patient_days.present? %>
-          <td class="type-percent" data-html="true" data-toggle="tooltip" data-placement="top" data-trigger="hover focus click"
-              data-template="<%= render "wide_tooltip_template" %>"
-              data-original-title="<%= render "drug_stocks_tooltip", report: @report.dig(:total_patient_days, drug_category) %>"
-              data-sort-column-key=<%= "#{drug_category}_patient_days" %>>
-            <em class="<%= patient_days_css_class(patient_days) %>">
-              <%= patient_days %>
-            </em>
-          </td>
-        <% else %>
-          <td class="type-blank"><span>&#8212;</span></td>
+        <% unless drug_category == "diabetes" %>
+          <% if patient_days.present? %>
```

Comment:
[nit] `if show_patient_days?(drug_category) && patient_days.present?`

Reply:
That's what I thought first, but the `patient_days.present?` conditional has an `else` below.